contributing.md

```diff
@@ -19,34 +19,36 @@ As a general rule all code changes are first committed to a branch. These branch
 ## Step by step
 
 ### Fork scenario
+
 1. Create a fork on github
 2. Clone your newly created fork `git clone https://github.com/<your account>/OpenDA.git`
 3. Create a branch with a name that describes your intended developments. `git checkout -b my_branch`
 4. Write and test your code in your branch.
 5. Commit to your local copy and push to your fork of OpenDA
-   * `git add bla`
-   * `git commit -m "my description"`
-   * `git push origin my_branch`
+    * `git add bla`
+    * `git commit -m "my description"`
+    * `git push origin my_branch`
 6. Once finished or more regularly: merge changes in OpenDA into your work
-  * Update your fork of OpenDA on github
-  * `git fetch origin master` to update your local copy (from your fork)
-  * `git merge master`
+    * Update your fork of OpenDA on github
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[list-item-content-indent] Don’t use mixed indentation for children, remove 1 space](https://app.codacy.com/gh/OpenDA-Association/OpenDA/pullRequest?prid=4902592)